torch/_inductor/ops_handler.py

```diff
@@ -664,3 +664,18 @@ def count(val):
 
 def _typecheck_OpCounterCSE(h: OpCounterCSE) -> OpsHandler[str]:
     return h
+
+
+class ExtractConstantsHandler(KernelFormatterHandler):
```

Comment:
Why do you inherit from `KernelFormatterHandler`?

Reply:
This is a workaround for `MockHandler`'s exponential blow-up issue. Basically if you do `x = ops.add(x, x)` repeatedly it expands to a single expression with 2^n elements.

Would be better to add a new ops handler base class that doesn't do any string formatting, but this would be a different change.

Reply:
Do you mind doing that refactoring? This is too hacky otherwise.